test_checker.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <assert.h>
+#include "checker.hpp"
+#include <string>
+
+int main() {
+    bool result = bBatteryIsOk(25, 70, 0.7);

```

Comment:
Test cases need to be strengthened further. Permutation and combinations of all three parameters should be considered. For example one parameter will have 2 boundary values and 3 in between values, i.e. <, ==, > threshold values. Hence a combination of all values for all parameters.